.gitignore

```diff
@@ -1,2 +1,3 @@
 *.egg-info
 __pycache__
+/.vscode
```

Comment:
No editor files in .gitignore please (protip: use [.gitignore_global](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) for stuff like this!)